src/renderer/api/kube-api.ts

```diff
@@ -93,9 +93,11 @@ export function ensureObjectSelfLink(api: KubeApi, object: KubeJsonApiData) {
   }
 }
 
-type KubeApiWatchOptions = {
+export type KubeApiWatchCallback = (data: IKubeWatchEvent, error: any) => void;
```

Comment:
```suggestion
export type KubeApiWatchCallback = (response: {data: IKubeWatchEvent, error?: any}) => void;
```

Reply:
I'm not sure why this suggestion 🤔

Reply:
I think it is so that you don't have to specify `null` when producing an error. Also is the standard order `(error: any, data: IKubeWatchEvent)`?

Reply:
In general I would propose always to use first argument as plain object.

_Benefits:_ 

- allows to pass args in any order
- better extensibility when overriding a method
- easier to collect and pass bunch of params with object's destruct/rest, e.g. `func({param = "default"} = {})`